### PR TITLE
Add contributor path page

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

Final page move from ansible/community-docs

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs/docsite/rst/community/contributor_path.rst
docs/docsite/rst/community/how_can_I_help.rst
docs/docsite/rst/community/index.rst
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/community/contributor_path.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
+****************
+Contributor path
+****************
+
+This section describes the contributor journey from the early beginning, through all the stages to becoming a leader who helps shape the future of Ansible. You can use this path as a roadmap for your long-term participation.
+
+Any contribution to the project, even a small one, is very welcome and valuable. Any contribution counts, whether it's feedback on an isssue, pull requiest, or  topic, a documentation change, or a coding contribution. When you contribute regularly, your proficiency and judgment in the related area are increasing fast and, along with this, the importance of your presence in the project.
```
Comment (ugwutotheeshoes) [inline docs/docsite/rst/community/contributor_path.rst]:
```suggestion
Any contribution to the project, even a small one, is very welcome and valuable. Any contribution counts, whether it's feedback on an issue, a pull request, a topic or documentation change, or a coding contribution. When you contribute regularly, your proficiency and judgment in the related area increase and, along with this, the importance of your presence in the project.
```

File: docs/docsite/rst/community/contributor_path.rst [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+.. contents::
+    :local:
+
+
+
+Determine your area of interest
+=================================
+
+First, determine areas that are interesting to you, taking into consideration your current experience and experience you'd like to gain. For example, if you use a certain collection, have a look there. See :ref:`how_can_i_help` for more ideas on how to help.
```
Comment (ugwutotheeshoes) [inline docs/docsite/rst/community/contributor_path.rst]:
```suggestion
First, determine areas that are interesting to you. Consider your current experience and what you'd like to gain. For example, if you use a specific collection, have a look there. See :ref:`how_can_i_help` for more ideas on how to help.
```

File: docs/docsite/rst/community/contributor_path.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+Find the corresponding the project
```
Comment (ugwutotheeshoes) [inline docs/docsite/rst/community/contributor_path.rst]:
```suggestion
Find the corresponding project
```

File: docs/docsite/rst/community/contributor_path.rst [hunk 4]
```diff
@@ -0,0 +1,104 @@
+****************
+Contributor path
+****************
+
+This section describes the contributor journey from the early beginning, through all the stages to becoming a leader who helps shape the future of Ansible. You can use this path as a roadmap for your long-term participation.
```
Comment (ugwutotheeshoes) [inline docs/docsite/rst/community/contributor_path.rst]:
```suggestion
This section describes the contributor journey from the beginning to becoming a leader who helps shape the future of Ansible. You can use this path as a roadmap for your long-term participation.
```

File: docs/docsite/rst/community/contributor_path.rst [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+- `Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_
+- `AWX <https://github.com/ansible/awx>`_
+- `Galaxy <https://galaxy.ansible.com/>`_
+- `ansible-lint <https://ansible-lint.readthedocs.io/en/latest/>`_
+- `Molecule <https://molecule.readthedocs.io/en/latest/>`_
+
+Learn
+=====
+
+The required skillset depends on the area of interest and the project you'll be working on. Remember that the best way to learn is by doing.
```
Comment (ugwutotheeshoes) [inline docs/docsite/rst/community/contributor_path.rst:34]: "Remember that the best way to learn is by doing."
is this line complete?

File: docs/docsite/rst/community/contributor_path.rst [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+Each Ansible project has its own set of contributor guidelines. Familarize yourself with these as you prepare your first contributions.
+
+* :ref:`Ansible Core development <developer_guide>`.
+* :ref:`Ansible collectoin development <developing_collections>` and the collection-level contributor guidelines in the collection repository.
+
+
+Making your first contribution
+==============================
+
+You can find some ideas how you can contribute in :ref:`how_can_i_help` and the `collection repository <https://github.com/ansible-collections/>`_  ``README`` and ``CONTRIBUTING`` files. To make your first experience as smooth as possible, read the repository documentation carefully, then ask the repository maintainers for guidance if you have any questions.
```
Comment (ugwutotheeshoes) [inline docs/docsite/rst/community/contributor_path.rst]:
```suggestion
You can find some ideas on how you can contribute in :ref:`how_can_i_help` and the `collection repository <https://github.com/ansible-collections/>`_  ``README`` and ``CONTRIBUTING`` files. To make your first experience as smooth as possible, read the repository documentation carefully, then ask the repository maintainers for guidance if you have any questions.
```

File: docs/docsite/rst/community/contributor_path.rst [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+Teach others
+============
+
+Share your experience with other contributors through `improving documentation <ommunity_documentation_contributions>`, answering question from other contributors and users  on `Matrix/Libera.Chat IRC <communicate>`, giving advice in issues and pull requests, and discussing the `Community Topics <https://github.com/ansible-community/community-topics/issues>`_.
+
+Become a collection maintainer
+===============================
+
+If you are a code contributor to a collection, you can get extended permissions in the repository and become a maintainer. A collection maintainer is a contributor trusted by the community who makes significant and regular contributions to the project and showed themselves as a specialist in the related area. See :ref:`maintainers` for details.
```
Comment (ugwutotheeshoes) [inline docs/docsite/rst/community/contributor_path.rst:80]:
```suggestion
If you are a code contributor to a collection, you can get extended permissions in the repository and become a maintainer. A collection maintainer is a contributor trusted by the community who makes significant and regular contributions to the project and serves as a specialist in the related area. See :ref:`maintainers` for details.
```

File: docs/docsite/rst/community/contributor_path.rst (hunk 2 above)
Comment (Andersson007) [reply docs/docsite/rst/community/contributor_path.rst]: @ugwutotheeshoes thank for the suggestion!
- Let's maybe split the sentence into 2 `First, determine areas that are interesting to you. Consider your current experience and what you'd like to gain.` ?

File: docs/docsite/rst/community/contributor_path.rst (hunk 3 above)
Comment (Andersson007) [reply docs/docsite/rst/community/contributor_path.rst]: good catch, thanks!

File: docs/docsite/rst/community/contributor_path.rst (hunk 5 above)
Comment (Andersson007) [reply docs/docsite/rst/community/contributor_path.rst:34]: Looks complete to me, i.e. "Keep it in mind that the best way to learn is by doing."

File: docs/docsite/rst/community/contributor_path.rst (hunk 7 above)
Comment (Andersson007) [reply docs/docsite/rst/community/contributor_path.rst:80]: i would keep the sentence as is.
`showed themselves` is an important thing here.
We're trying to say that "the person's judgment and activities (like great reviews, for example) show that they are a specialist in the technical area the collection servers. So we can trust their judgment and promote them to a maintainer."

File: docs/docsite/rst/community/contributor_path.rst (hunk 2 above)
Comment (ugwutotheeshoes) [reply docs/docsite/rst/community/contributor_path.rst]: OK, That's better that way.
Sorry about the late reply too.

Comment (samccann) [reply docs/docsite/rst/community/contributor_path.rst]: I've updated the suggested edit to match this idea. @ugwutotheeshoes- I think in the future you have the ability to edit your own suggested edits as well. Just click `...` to the top right of the suggestion and it should give you the ability to edit. I'll be doing that here now so I can merge the changes. and thanks!

File: docs/docsite/rst/community/contributor_path.rst (hunk 7 above)
Comment (samccann) [reply docs/docsite/rst/community/contributor_path.rst:80]: agreed.

**Comment by Andersson007** [comment]
@samccann @ugwutotheeshoes thank you for working on this!

**Pull request merged**